packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -123,67 +123,77 @@ const COLUMNS = [
         vms: getAllVms(state, props),
         vbds: getVbds(state, props),
       })
-    })(({ vbds, vms }) => {
-      if (isEmpty(vms)) {
+    })(({ item: vdi, vbds, vms, userData: { vmsSnapshotsBySuspendVdi } }) => {
+      const vmSnapshot = vmsSnapshotsBySuspendVdi[vdi.uuid]
+
+      if (vmSnapshot === undefined) {
```

Comment:
I don't get this, won't this be true for most of the VDIs? What do we want to test here?

Reply:
Ho, I did a code suggestion for this problem, and I completely missed it during my last review.
https://github.com/vatesfr/xen-orchestra/pull/7391#discussion_r1502348824